src/app/features/auth/signup/SignUpForm.tsx

```diff
@@ -72,11 +73,18 @@ export const SignUpForm: FC<{
                 />
               </Col>
             </Row>
+            <FormControl
+              label={labels.form.phone.label}
+              name="phone"
+              autoComplete="username-phone"
```

Comment:
i believe the correct one is space separated

Reply:
Fixed